stylecop/GlobalSupressions.Test.cs

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
We already have a `Shared` folder with stylecop files. Should we create a folder under MSALWrapper.Test and move test stylecop stuff there instead of keeping it at src level?

Reply:
Discussed offline - moved the stylecop folder into src.